src/analyzer.ts

```diff
@@ -908,31 +833,25 @@ const assertArray = (
  * Throw an error if it was an object literal ({...}), assuming it was parsed
```

Comment:
Can be another PR, but just noticed a bunch of these functions now have incorrect jsdoc, since they don't throw anymore.